course02/homework01/danielellis/linkedlist.cc

```diff
@@ -0,0 +1,86 @@
+#include "linkedlist.h"
+
+#include <stdexcept>
+
+using namespace std;
```

Comment:
This isn't good practice. As soon as you will use other libs like boost, you will get conflicts.